implementation/src/main/java/io/smallrye/config/lifecycle/ConfigValue.java

```diff
@@ -0,0 +1,39 @@
+package io.smallrye.config.lifecycle;
```

Comment:
Yeah, this was with the other first implementation. The new package was also to make it easier to follow the changes. Since we are now coming to a more final solution, we can just move them back to the main one.